packages/desktop/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -63,7 +63,7 @@
     let outputOptions: OutputOptions
```

Comment:
I think you're not getting the timelockDate from the `newTransactionDetails` here. It doesn't show in the confirmation page.

Reply:
the `SendConfirmationPopup` is part of the old send flow. Currently there is no way to set a timelock in that flow

Reply:
Then my comment is put on the wrong place, but I still have the issue that if I set a time lock, I don't see it on the confirmation page. Tbh, this can go into a separate PR.

Reply:
https://github.com/iotaledger/firefly/issues/6819